package.json

```diff
@@ -388,6 +388,12 @@
                     ".xquery"
                 ],
                 "configuration": "./xquery-ml.configuration.json"
+            },
+            {
+                "id": "javascript",
```

Comment:
Is this telling VSCode - hey, when you use this plugin, use the existing "javascript" language for files ending in ".sjs"?

Reply:
It's just associating the .sjs file extension with that language ID (which is the ID for the built-in JavaScript language). So yes, except that there's no requirement that the language has to already exist. It just happens to in this case.